packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -779,14 +779,15 @@ export default class LocusInfo extends EventsScope {
   /**
    * @param {Object} info
    * @param {Object} self
+   * @param {string} meetingState
    * @returns {undefined}
    * @memberof LocusInfo
    */
-  updateMeetingInfo(info, self) {
+  updateMeetingInfo(info, self, meetingState) {
     if (info && (!isEqual(this.info, info))) {
       const roles = self ? SelfUtils.getRoles(self) : this.parsedLocus.self?.roles || [];
 
-      const parsedInfo = InfoUtils.getInfos(this.parsedLocus.info, info, roles);
+      const parsedInfo = InfoUtils.getInfos(this.parsedLocus.info, info, roles, meetingState === MEETING_STATE.STATES.JOINED);
```

Comment:
question: Is this the actual constant you need here? There are two constants that both correspond to the string "JOINED". In the selfUtils the comparisons against state are made using the underscores _JOINED_